mbedtls/Cargo.toml

```diff
@@ -40,6 +40,10 @@ default-features = false
 features = ["custom_printf", "trusted_cert_callback", "threading"]
```

Comment:
I guess technically `custom_printf` is not needed here anymore.

Reply:
Removed